lib/validators/pop3_polling_enabled_setting_validator.rb

```diff
@@ -33,6 +33,7 @@ def error_message
   private
 
   def authentication_works?
+    return true if Rails.env.test?
```

Comment:
I prefer to stub `authentication_works?` methods in tests instead of having that line, but I know that we have few of Rails.env.test? in codebase so I guess it is a personal preference

Reply:
I realised I was no longer testing this anyway, so I removed this line :)